snooty/parser.py

```diff
@@ -725,6 +726,10 @@ def handle_directive(
                 return doc
 
             if api_version:
+                if api_version != "2.0":
```

Comment:
we have a mechanism in rstspec.toml called `enum` to enumerate allowed values for an option; it's unfortunate that we don't currently allow using that up to directive arguments

Reply:
Oh coooool!!! I will switch! Thanks, I'll let you know when I've re-configured for a quick glance!

Reply:
Oh no, you'd have to change some stuff in the parser -- it's a flaw that it doesn't work as one would expect 😅  This is okay as-is for now unless you actually want to go mucking around!

Reply:
oh! Just saw this. I just pushed up using enums, and it seems to work?? But let me know if I should revert!

Reply:
I also deleted an unused enum